gcsfs/tests/test_core.py

```diff
@@ -80,9 +80,16 @@ def test_list_bucket_multipage(token_restore):
 
 @my_vcr.use_cassette(match=['all'])
 def test_pickle(token_restore):
+    import pickle
     with gcs_maker() as gcs:
-        import pickle
-        gcs2 = pickle.loads(pickle.dumps(gcs))
+        gcs['abcdefg'] = b'1234567'
```

Comment:
`gcs` is a GCSFileSystem, here, not a mapping. This should have been a new pickle test in `test_mapping.py`?

Reply:
Oh, I see you have that below - then this should not have been changed, I think.

Reply:
Right.  My mistake.  I've altered this test to use the traditional `open` interface.